README.md

```diff
@@ -16,7 +16,20 @@
 
 ## Background
 
-TBD
+This is primarily intended for app authorization (for applications using the
+`client_credentials` OAuth 2 grant type), rather than for user (`sub`ject)
```

Comment:
Is "user (`sub`ject)" a typo?

Reply:
I'm thinking no, because I was able to google "`aud`ience claim" below and see that "aud" is used as the key.

Reply:
Hehe yeah, `aud` and `sub` are registered JWT claims.